libs/transactions/src/lib/services/transactions-journey-config.service.ts

```diff
@@ -1,7 +1,8 @@
-import { Injectable } from '@angular/core';
+import { Injectable, TemplateRef } from '@angular/core';
 
 @Injectable()
 export class TransactionsJourneyConfiguration {
   pageSize = 20;
   slimMode = true;
-}
+  additionalDetailsTpl: TemplateRef<any> | undefined = undefined;
```

Comment:
Should we add a type for the context the template uses for rendering?
Or is this overkill, and will `any` suffice?

Reply:
added typing in new commit